test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -654,6 +671,7 @@ def _ensure_imports_below_docs(self, doc_info, first_callable):
                         if future_import.name not in self.WHITELIST_FUTURE_IMPORTS:
                             self.reporter.error(
                                 path=self.object_path,
+                                name='allowed-future-imports',
```

Comment:
This test should be removed from the module validator. We have a future boilerplate sanity check which covers this. I'd remove it in a separate PR though.